cli/common.py

```diff
@@ -53,6 +53,20 @@ def func_wraper(*args, **kwargs):
     return func_wraper
 
 
+def check_terminal_utf8_support():
+    """
+    Checks weather the terminal supports UTF-8 glyphs.
```

Comment:
s/weather/whether/

Reply:
ah, I never remember how to write that word correctly :-)